### DISCUSSION TITLE
插件如何上架 DevEco studio插件市场

### DISCUSSION THREAD

**Discussion by AutumnFish** [YiiGuxing/TranslationPlugin#4981] [Ideas]
请问如何将自己开发的插件上架到 DevEco studio的市场里面
  最近自己在编码的时候写了个小工具，在官方论坛里面只找到了开发插件的方式，找不到如何上线
  看到翻译插件可以上线DevEco的市场，所以来请教一下 🙏

**Comment by YiiGuxing** [comment]
DevEco studio 我没使用过，不知道它有没有自己的插件市场， 不过如果它能够使用JetBrains的插件市场的话，你可以发布到[JetBrains的插件市场](https://plugins.jetbrains.com)。

**Comment by AutumnFish** [comment]
嗯嗯好的，感谢回复～
因为看到在DevEco studio里面最近在用翻译插件，看到是你开发的，以为是需要申请或者提交什么，就想着请教一下
那这么看的话应该是DevEco那边选择了一些优质的插件直接迁移过去了👍


时间煮鱼`
***@***.***



&nbsp;




------------------&nbsp;原始邮件&nbsp;------------------
发件人:                                                                                                                        "YiiGuxing/TranslationPlugin"                                                                                    ***@***.***&gt;;
发送时间:&nbsp;2023年12月16日(星期六) 晚上8:28
***@***.***&gt;;
***@***.******@***.***&gt;;
主题:&nbsp;Re: [YiiGuxing/TranslationPlugin] 插件如何上架 DevEco studio插件市场 (Discussion #4981)





 
DevEco studio 我没使用过，不知道它有没有自己的插件市场， 不过如果它能够使用JetBrains的插件市场的话，你可以发布到JetBrains的插件市场。
